src/components/DatePicker/DatePicker.test.js

```diff
@@ -8,25 +7,40 @@ describe('<DatePickerStyled /> rendering', () => {
   const setState = jest.fn();
 
   it('should render without crashing', () => {
-    const component = shallow(
-      <DatePickerStyled name="test" date={new Date('Thu Nov 12 2020 00:00:00 GMT-0000')} setState={setState} />
-    );
+    const { component } = shallowWithProviders(
+      <DatePickerStyled
+        name="test"
+        dateFormat="en-US"
+        date={new Date('Thu Nov 12 2020 00:00:00 GMT-0000')}
+        setState={setState}
+      />
+    )({ preferences: { locale: 'en' } });
```

Comment:
`shallowWithProviders` already adds the locale 'en' to the initial state of the store.